Test/DafnyTests/DafnyTests.cs

```diff
@@ -0,0 +1,289 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using Xunit;
+using Xunit.Sdk;
+using YamlDotNet.RepresentationModel;
+using YamlDotNet.Serialization;
+
+namespace DafnyTests {
+
+    public class DafnyTests {
```

Comment:
Use 2-space indentation.

Reply:
Can do. The implementation in general needs cleaning up before the non-draft PR.